js/data/array_store.js

```diff
@@ -76,8 +76,8 @@ var ArrayStore = Store.inherit({
         });
     },
 
-    _byKeyImpl: function(key) {
-        var index = this._indexByKey(key);
+    _byKeyImpl: function(key, extraOptions, cache) {
```

Comment:
Where the extraOptions parameter use?

Reply:
https://github.com/DevExpress/DevExtreme/search?q=extraOptions&type=Code

Reply:
The extraOptions variable is unused in this method.

Reply:
But extraOptions parameter is used in ODataStore.